sw/airborne/subsystems/radio_control/sbus_common.h

```diff
@@ -58,9 +58,11 @@
  * depending of the RC transmitter.
  * The radio XML file is used to assign the
  * input values to RC channels.
+ * 4MS timeout
  */
 #define SBUS_BUF_LENGTH 24
 #define SBUS_NB_CHANNEL 16
+#define SBUS_TIMEOUT_MS 4
```

Comment:
Why is this one in the header? If not needed outside of sbus_common.c, can you move it to the source file?

It might also be nice to have this configurable, maybe the delay can change depending on the application. If you add the setting, also add it to the module description.

Reply:
Not really needed to be configurable, since the baudrate of SBUS and the frame length are defined this timeout does never change